Engine/UserInput.cs

```diff
@@ -317,7 +316,12 @@ bool isBrowsable(IMemberData m)
                     else if(string.IsNullOrEmpty(str.Value) == false)
                         Console.Write($"({str.Value}): ");
                     else Console.WriteLine(":");
-                    
+
+                    if (Console.IsInputRedirected)
+                    {
+                        log.Warning("\nStandard Input is redirected. Default option chosen.\n");
```

Comment:
I don't think we should emit a warning. Whatabout writing something a bit more useful as an Info message.

e.g "Selecting '{value}' for '{property}'?